mewarpx/mewarpx/diags_store/field_diagnostic.py

```diff
@@ -47,6 +49,7 @@ def __init__(self, diag_steps, diag_data_list, grid, name, write_dir,
         self.write_dir = write_dir
         self.plot_on_diag_step = plot_on_diag_step
         self.plot_data_list = plot_data_list
+        self.plot_species_list = plot_species_list
```

Comment:
Could you please make this behave as follows (suggestion is untested so please confirm that it works)
```suggestion
        if plot_species_list is not None:
            self.plot_species_list = plot_species_list
        else:
            self.plot_species_list = [species.name for species in mwxrun.species]
```